packages/client/src/object/Cache.test.ts

```diff
@@ -396,8 +390,8 @@ describe("AsyncCache", () => {
               ]);
 
               itReturnsForAsyncGet([
-                expect.any(Promise),
-                expect.any(Promise),
+                undefined,
```

Comment:
Before we could get an object that appeared to be a Promise from `.results` when the object was result was pending. So these `excpect.any(Promise)` would technically work. However now they split the settledResults out and as those first two promises are still pending, the result is undefined in vitest.